### PR TITLE
ocp: make export token TTL configurable

### PR DESCRIPTION
**Pull request by bennyz**

### CODE REVIEW

**Comment by bennyz** [comment]
> I think this granularity (a global configuration) is good enough are we good with CDI's default setting?

well the docs say it's a reasonable default :)
```
	// ttlDuration limits the lifetime of an export
	// If this field is set, after this duration has passed from counting from CreationTimestamp,
	// the export is eligible to be automatically deleted.
	// If this field is omitted, a reasonable default is applied.
	// +optional
```

File: pkg/controller/plan/adapter/ocp/client.go [hunk 1]
```diff
@@ -143,23 +143,31 @@ func (r *Client) PreTransferActions(vmRef ref.Ref) (ready bool, err error) {
...
 				Kind:       "VirtualMachineExport",
 				APIVersion: "kubevirt.io/v1alpha3",
 			},
-			ObjectMeta: meta.ObjectMeta{
+			ObjectMeta: metav1.ObjectMeta{
 				Name:      vmRef.Name,
 				Namespace: vmRef.Namespace,
 			},
 			Spec: export.VirtualMachineExportSpec{
+				TTLDuration: tokenTTLDuration,
```
Comment (ahadas) [inline pkg/controller/plan/adapter/ocp/client.go:170]: is it omittedwhen `tokenTTLDuration` is nil?

Comment (bennyz) [reply pkg/controller/plan/adapter/ocp/client.go:170]: indeed

Comment (ahadas) [reply pkg/controller/plan/adapter/ocp/client.go:170]: ack

**Review by ahadas — APPROVED** [review]

**Pull request merged**